libbeat/docs/generalconfig.asciidoc

```diff
@@ -121,13 +99,13 @@ fields:
 
 ===== refresh_topology_freq
 
-The refresh interval of the topology map in
+DEPRECATED: The refresh interval of the topology map in
```

Comment:
We have special asciidoc syntax for deprecations, which makes it more likely that we'll remember to cleanup the docs after removal. See [here](https://github.com/elastic/docs#section-notifications).